hedy.py

```diff
@@ -2563,10 +2563,87 @@ def create_grammar(level, lang="en"):
         grammar_text_i = get_additional_rules_for_level(i)
         result = merge_grammars(result, grammar_text_i, i)
 
+    # Change the grammar if skipping faulty is enabled
+    if source_map.skip_faulty:
```

Comment:
I just have a quick question! Skipping faulty is enabled as an option in Hedy or is enabled when the user makes a mistake?

Reply:
This is an option in Hedy! The user posts a request with _skip_faulty_ enabled (true) or disabled (false). 
When the user makes a mistake (the first time) the error is simply returned, after that, the client posts the code again with _skip_faulty_  set to true. I hope this answers your question.